app/src/main/java/com/example/android_receipt_app/ReceiptEntity.kt

```diff
@@ -3,6 +3,7 @@ package com.example.android_receipt_app
 data class ReceiptEntity(
         val title: String?,
         val image: String?,
-        val description: String?
+        val description: String?,
+        val creationDate: String?
 )
```

Comment:
5. Part 1/2:

We want to have real date of adding receipt here.
To achieve it, change the String type of creationDate into Date type.